.circleci/config.yml

```diff
@@ -560,6 +560,7 @@ jobs:
             if [ -n "${INSTANCE_TESTS}" ] && [ $CIRCLE_NODE_INDEX -ne 1 ] ;
               then
                 ./Tests/scripts/instance_test.sh
+                cp ./Tests/failed_instances.txt $CIRCLE_ARTIFACTS/failed_instances.txt
```

Comment:
this is hiding the error code of the bash script, need to handle it like we do [here](https://github.com/demisto/content/pull/7624/files#diff-1d37e48f9ceff6d8030570cd36286a61R530-R532)

Reply:
Is this an escape room? link leads to nowhere...

Reply:
see lines 530-532 in this file.